### ISSUE TITLE
Feedback: Choosing an appropriate model

### ISSUE THREAD

**Issue by CarmenTamayo** [epiverse-trace/tutorials#63]
1. •	I'm not 100% sure why the pre-requisite of this episode is to have previously completed the simulating tx episode, especially since the latter is so much longer and complex, are these episodes building on the previous one? My understanding is that they are bite-sized, standalone tutorials, or at least that's what transpires when reading the Summary  and setup section. If this is the case, this should be explicitly said in this section. Further, by reading the contents of this episode, I'm not sure that previously completing "simulating tx" is very necessary, if anything I'd argue that it is actually the other way around since, as I commented on issue #60, on simulating tx the models in the {epidemics} library are mentioned, as well as an indication that it is important to check the model specifications, but there's no guidance on how to do so on that episode, which is available on the "choosing the appropriate model" episode. I believe the pre-requisites here should be more about the theory behind ID modelling, or about why there's a need for modelling in outbreak response and analysis.
2. •	In the "Overview" section, I'd rephrase the question to "How do I choose a mathematical model that's appropriate/optimal to complete my analytical task?", and I'd also change the second objective to "Understand the model requirements for a specific research question" or "for a specific analytical step".
3. •	"Using mathematical models in outbreak analysis does not necessarily require developing a new model" <- I think this sentence should be removed, as it somehow assumes that learners will have the pre-conceived idea that models need to be developed from scratch in every case, which I'm not sure is true, and it can create more confusion than good.
4. •	"In this tutorial, we will learn how to choose an existing model to generate predictions for a given scenario." I think it'd be very important to clarify the types of modelling according to the purpose of the analysis in epi/outbreak analysis, e.g., the difference between forecasts/scenario modelling/retrospective analysis, either as part of the tutorial or as pre-requisite and clarify which one is intended in this episode. The episode goes into detail about stochastic vs deterministic, but doesn't mention this consideration, which I believe would be very significant for learners, especially for epidemiologists or clinicians new to modelling. Since the episode is quite short, and the purpose is to guide readers on choosing a modelling approach, I think a section about model purposes/applications would be a valuable addition. 
5. •	"When deciding whether an existing model can be used" -> it'd be better to say something like "When deciding which mathematical model to use" or "which would be most appropriate" 
6. •	In "what is the outcome of interest", I would either provide a more extensive list of outcomes or provide a reference to further reading.
7. •	In the section "will any interventions be modelled", I'd either remove the sentence "We will discuss interventions in detail in the next tutorial." or include a link to said tutorial (or better said, episode) instead.
8. •	Overall I feel like the challenge in this particular episode is a bit short and perhaps a bit "too easy", i.e., we're telling the readers that there are 2 models to choose from, and one is a generic one and the other one is for ebola, and the question is about modelling ebola, so this is not very challenging. The challenge doesn't involve a practical exercise- I'm not saying this is 100% necessary, but it would be nice for readers to actually *do* something, for instance, this challenge could include practical guidance on how to check the characteristics of the models included in {epidemics}- right now it just says that students can read the documentation of the package and then a reminder about checking model equations, but there is no practical example of this process, or how to link this process with the challenge- so the actual challenge seems a bit disconnected from the rest of the text. Adding a second, perhaps more difficult question with another outbreak scenario would be nice. It might also be helpful to include a further section where the question is actually solved using the package {epidemics}, although this might not be necessary if similar exercises are conducted in other episodes (then I'd reference said episode).
9. •	The key points of this episode are also a bit vague and don’t really highlight the message conveyed in this tutorial. Maybe the first one could be about how existing mathematical models should be selected according to the research questions/analytical tasks required by the user, and how there is a trusted process to do so that is demonstrated in the episode. The second one could say that it is important to check…. As right now it sounds more like an imperative than a key point.

**Comment by amanda-minter** [comment]
> 4. •	"In this tutorial, we will learn how to choose an existing model to generate predictions for a given scenario." I think it'd be very important to clarify the types of modelling according to the purpose of the analysis in epi/outbreak analysis, e.g., the difference between forecasts/scenario modelling/retrospective analysis, either as part of the tutorial or as pre-requisite and clarify which one is intended in this episode. The episode goes into detail about stochastic vs deterministic, but doesn't mention this consideration, which I believe would be very significant for learners, especially for epidemiologists or clinicians new to modelling. Since the episode is quite short, and the purpose is to guide readers on choosing a modelling approach, I think a section about model purposes/applications would be a valuable addition.

I think this could be added to the introduction episode #67

**Comment by avallecam** [comment]
> > 4. •	"In this tutorial, we will learn how to choose an existing model to generate predictions for a given scenario." I think it'd be very important to clarify the types of modelling according to the purpose of the analysis in epi/outbreak analysis, e.g., the difference between forecasts/scenario modelling/retrospective analysis, either as part of the tutorial or as pre-requisite and clarify which one is intended in this episode. The episode goes into detail about stochastic vs deterministic, but doesn't mention this consideration, which I believe would be very significant for learners, especially for epidemiologists or clinicians new to modelling. Since the episode is quite short, and the purpose is to guide readers on choosing a modelling approach, I think a section about model purposes/applications would be a valuable addition.
> 
> I think this could be added to the introduction episode #67

I added this in a new issue #94 (this fits into the potential plan to isolate episodes per task to discuss in #69 )
